client/js/components/App.react.jsx

```diff
@@ -39,6 +73,37 @@ export default class App extends React.Component {
             .tap((data) => this.setState({trailGeo: data}));
     }
 
+    setUserLocation(userLocation) {
+        var parksWithDistance = this.state.allParks.map((park) => {
+            var toPoint = turf.point([userLocation[1], userLocation[0]]);
+            var fromPoint = turf.point([park.center[1], park.center[0]]);
```

Comment:
When we forget to swap everything breaks in a very visible way.

Turf works with GeoJSON, so things `follow x, y, z order (easting, northing, altitude`.

So, its not really possible to tell it the data is backwards (since then it would be altitude, northing, easting). :D